crates/starknet-devnet-core/src/starknet/mod.rs

```diff
@@ -1211,6 +1221,10 @@ mod tests {
             starknet.pending_block().header.l1_gas_price.price_in_wei,
             GasPrice(initial_gas_price.get())
         );
+        assert_eq!(
+            starknet.pending_block().header.l1_data_gas_price.price_in_wei,
+            GasPrice(initial_data_gas_price.get())
+        );
         assert_eq!(starknet.pending_block().header.sequencer.0, initial_sequencer);
```

Comment:
Should there be two assertions?

Reply:
modified in [2c97237](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/392/commits/2c97237669fd371e7fabb910e42493f6e530cf17)